ui/pages/SignIn.tsx

```diff
@@ -81,90 +85,130 @@ function SignIn() {
 
   const handleUserPassSubmit = () => signIn({ username, password });
 
+  const authOptions = (
+    <>
+      {error && (
+        <Alert
+          severity="error"
+          title="Error retrieving auth setup details"
+          message={String(error)}
+          center
+        />
+      )}
+      {flags.OIDC_AUTH ? (
+        <Flex wide center>
+          <Button
+            type="submit"
+            onClick={(e) => {
+              e.preventDefault();
+              handleOIDCSubmit();
+            }}
+          >
+            LOGIN WITH OIDC PROVIDER
+          </Button>
+        </Flex>
+      ) : null}
+      {flags.OIDC_AUTH && flags.CLUSTER_USER_AUTH ? (
+        <Divider variant="middle" style={{ margin: theme.spacing.base }} />
+      ) : null}
+      {flags.CLUSTER_USER_AUTH ? (
+        <form
+          ref={formRef}
+          onSubmit={(e) => {
+            e.preventDefault();
+            handleUserPassSubmit();
+          }}
+        >
+          <Flex center align>
+            <Input
+              onChange={(e) => setUsername(e.currentTarget.value)}
+              id="email"
+              type="text"
+              placeholder="Username"
+              value={username}
+            />
+          </Flex>
+          <Flex center align>
+            <Input
+              onChange={(e) => setPassword(e.currentTarget.value)}
+              required
+              id="password"
+              placeholder="Password"
+              type={showPassword ? "text" : "password"}
+              value={password}
+              endAdornment={
+                <InputAdornment position="end">
+                  <IconButton
+                    aria-label="toggle password visibility"
+                    onClick={() => setShowPassword(!showPassword)}
+                  >
+                    {showPassword ? <Visibility /> : <VisibilityOff />}
+                  </IconButton>
+                </InputAdornment>
+              }
+            />
+          </Flex>
+          <Flex center>
+            {!authLoading ? (
+              <Button type="submit" style={{ marginTop: theme.spacing.medium }}>
+                CONTINUE
+              </Button>
+            ) : (
+              <div style={{ margin: theme.spacing.medium }}>
+                <LoadingPage />
+              </div>
+            )}
+          </Flex>
+        </form>
+      ) : null}
+    </>
+  );
+
   return (
-    <SignInPageWrapper tall wide center align column>
-      <FormWrapper center align wrap>
-        {error && (
-          <AlertWrapper
-            severity="error"
-            title="Error signin in"
-            message={`${String(error.status)} ${error.statusText}`}
-            center
-          />
-        )}
+    <Flex
+      tall
+      wide
+      center
+      align
+      column
+      style={{
+        height: "100vh",
+        width: "100vw",
+      }}
+    >
+      <Lottie
+        play
+        loop
```

Comment:
The animation doesn't loop very well, we could slow it down, bit more chill, seems to last ~20s+ in that case.

```suggestion
        loop={false}
        speed={0.3}
```

Reply:
```
        renderer="canvas"
```

Seems to be a bit more gentle on the CPU too but doesn't respect aspect ratio.